core/arch/x86/decode_table.c

```diff
@@ -5579,8 +5647,11 @@ const instr_info_t e_vex_extensions[][3] = {
     {INVALID, 0x663a1918, "(bad)", xx, xx, xx, xx, xx, no, x, NA},
   }, { /* e_vex ext 76 */
     {INVALID,   0x663a1d18, "(bad)", xx, xx, xx, xx, xx, no, x, NA},
-    {OP_vcvtps2ph, 0x663a1d18, "vcvtps2ph", Wx, xx, Vx, Ib, xx, mrm|vex|reqp, x, END_LIST},
-    {INVALID, 0x663a1d18, "(bad)", xx, xx, xx, xx, xx, no, x, NA},
+    {OP_vcvtps2ph, 0x663a1d18, "vcvtps2ph", Wx, xx, Vx, Ib, xx, mrm|vex|reqp, x, tvex[76][2]},
+    /* XXX i#3639, tools tend to give source/destination different register size mnemonic.
+     * This also affects the existing VEX version if it exists.
+     */
+    {OP_vcvtps2ph, 0x663a1d18, "vcvtps2ph", We, xx, KEw, Ve, Ib, mrm|evex|reqp, x, END_LIST},
```

Comment:
I'm still seeing problems:

```
  0x0000563ada68c066  62 f3 7d 08 1d 41 37 vcvtps2ph oword ptr [rcx+0x37] {k0}, xmm0, 0x25
```

```
Disassembling 0x62 0xf3 0x7d 0x08 0x1d 0x41 0x37
<stdin>:1:1: warning: invalid instruction encoding
0x62 0xf3 0x7d 0x08 0x1d 0x41 0x37
^
llvm-mc:   0x62 0xf3 0x7d 0x08 0x1d 0x41 0x37 
capstone: 
bfd:       62 .byte 0x62 1d .byte 0x1d
```

Reply:
Are you sure you're feeding the last byte into the disassembler? The 0x25 is the immediate.

echo '0x62 0xf3 0x7d 0x08 0x1d 0x41 0x37 0x25' | llvm-mc-7 --disassemble -triple=x86_64
.text
vcvtps2ph       $37, %xmm0, 440(%rcx)

Capstone has an issue with the instruction, missing support. Binutils + llvm are ok.

Reply:
The 25 on the next line was the problem.